src/unstract/sdk/tool/entrypoint.py

```
@@ -6,6 +9,14 @@
 class ToolEntrypoint:
     """Class that contains methods for the entrypoint for a tool."""
 
+    @staticmethod
+    def _signal_handler(signum, frame):
```

Ruff (0.11.9)

Missing return type annotation for staticmethod `_signal_handler`

Add return type annotation: `None`

(ANN205)